### PR TITLE
Explicit ref insights

### PR DESCRIPTION
**Pull request by gucalder**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@gucalder__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (gucalder). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/Insights/Microsoft.Azure.Insights.nuspec [hunk 1]
```diff
@@ -21,7 +21,10 @@
       </group>
     </references>
     <dependencies>
+      <dependency id="Hyak.Common" version="1.0.1" />
       <dependency id="Microsoft.Azure.Common" version="[2.0.1,3.0)" />
+      <dependency id="Microsoft.WindowsAzure.Storage" version="4.0.0" />
+      <dependency id="Newtonsoft.Json" version="6.0.4" />
```
Comment (markcowl) [inline src/Insights/Microsoft.Azure.Insights.nuspec]: Remove this dependency, The Newtonsoft.Json version comes from your dependencies

File: src/Insights/Microsoft.Azure.Insights.nuspec [hunk 2]
```diff
@@ -21,7 +21,10 @@
       </group>
     </references>
     <dependencies>
+      <dependency id="Hyak.Common" version="1.0.1" />
```
Comment (markcowl) [inline src/Insights/Microsoft.Azure.Insights.nuspec]: Remove this dependency, the Hyak.Common dependency comes from Azure.Common

File: src/Insights/Customizations/Sku/SkuOperationsExtensions.Customizations.cs [hunk 3]
```diff
@@ -0,0 +1,131 @@
+//
```
Comment (markcowl) [inline src/Insights/Customizations/Sku/SkuOperationsExtensions.Customizations.cs]: Isn't this just generated?  If so, this should go in the Generated directory

File: src/Insights/Customizations/Sku/ISkuOperations.Customizations.cs [hunk 4]
```diff
@@ -0,0 +1,80 @@
+//
```
Comment (markcowl) [inline src/Insights/Customizations/Sku/ISkuOperations.Customizations.cs]: If this file is generated, it should go in the 'Generated' folder

File: src/Insights/Customizations/Sku/SkuOperations.Customizations.cs [hunk 5]
```diff
@@ -0,0 +1,332 @@
+//
```
Comment (markcowl) [inline src/Insights/Customizations/Sku/SkuOperations.Customizations.cs]: Same comment

File: src/Insights/Generated/Management/Insights/AutomatedExportOperations.cs [hunk 6]
```diff
@@ -0,0 +1,1036 @@
+//
```
Comment (markcowl) [inline src/Insights/Generated/Management/Insights/AutomatedExportOperations.cs]: same comment

File: src/Insights/Generated/Management/Insights/AutomatedExportOperationsExtensions.cs [hunk 7]
```diff
@@ -0,0 +1,262 @@
+//
```
Comment (markcowl) [inline src/Insights/Generated/Management/Insights/AutomatedExportOperationsExtensions.cs]: same comment

File: src/Insights/Generated/Management/Insights/IAutomatedExportOperations.cs [hunk 8]
```diff
@@ -0,0 +1,126 @@
+//
```
Comment (markcowl) [inline src/Insights/Generated/Management/Insights/IAutomatedExportOperations.cs]: same comment

File: src/Insights/Generated/Management/Insights/ISkuOperations.cs [hunk 9]
```diff
@@ -0,0 +1,56 @@
+//
```
Comment (markcowl) [inline src/Insights/Generated/Management/Insights/ISkuOperations.cs]: same comment

File: src/Insights/Microsoft.Azure.Insights.nuspec (hunk 1 above)
Comment (gucalder) [inline src/Insights/Microsoft.Azure.Insights.nuspec]: Removed.

File: src/Insights/Customizations/Sku/SkuOperationsExtensions.Customizations.cs (hunk 3 above)
Comment (gucalder) [inline src/Insights/Customizations/Sku/SkuOperationsExtensions.Customizations.cs]: I just got confirmation that this is not generated.

File: src/Insights/Customizations/Sku/ISkuOperations.Customizations.cs (hunk 4 above)
Comment (gucalder) [inline src/Insights/Customizations/Sku/ISkuOperations.Customizations.cs]: I just got confirmation that this is not generated.

File: src/Insights/Customizations/Sku/SkuOperations.Customizations.cs (hunk 5 above)
Comment (gucalder) [inline src/Insights/Customizations/Sku/SkuOperations.Customizations.cs]: I just got confirmation that this is not generated.

File: src/Insights/Generated/Management/Insights/AutomatedExportOperations.cs (hunk 6 above)
Comment (gucalder) [inline src/Insights/Generated/Management/Insights/AutomatedExportOperations.cs]: This is already in the generated folder.

File: src/Insights/Generated/Management/Insights/AutomatedExportOperationsExtensions.cs (hunk 7 above)
Comment (gucalder) [inline src/Insights/Generated/Management/Insights/AutomatedExportOperationsExtensions.cs]: This is already in the generated folder.

File: src/Insights/Generated/Management/Insights/IAutomatedExportOperations.cs (hunk 8 above)
Comment (gucalder) [inline src/Insights/Generated/Management/Insights/IAutomatedExportOperations.cs]: This is already in the generated folder.

File: src/Insights/Generated/Management/Insights/ISkuOperations.cs (hunk 9 above)
Comment (gucalder) [inline src/Insights/Generated/Management/Insights/ISkuOperations.cs]: This is already in the generated folder.

File: src/Insights/Microsoft.Azure.Insights.nuspec (hunk 2 above)
Comment (gucalder) [inline src/Insights/Microsoft.Azure.Insights.nuspec]: Removed.

**Comment by markcowl** [comment]
@azuresdkci test this please

**Pull request merged**